bucket/featherwallet.json

```diff
@@ -0,0 +1,39 @@
+{
+    "version": "1.0.1",
+    "description": "A Monero SPV Wallet",
+    "homepage": "https://featherwallet.org/",
+    "license": {
+        "url": "https://git.featherwallet.org/feather/feather/raw/branch/master/LICENSE"
+    },
+    "architecture": {
+        "64bit": {
+            "url": "https://featherwallet.org/files/releases/win/feather-1.0.1-win.zip",
+            "hash": "aa5f0a38d6258af2deb01c5c5c35acbc1e1311882357e7996c3022bff48932fe"
+        }
+    },
+    "pre_install": [
+        "Move-Item \"$dir\\feather*.exe\" \"$dir\\feather.exe\"",
```

Comment:
What is the default name after unzipping?

feather*.exe could cause issues later if they release something else as well

Reply:
So if possible we should try to fix this to the best of our abilities.